docs/using-github-extensions.md

```diff
@@ -0,0 +1,30 @@
+# Using GitHub Apps, Extensions, and bots
+
+In OpenTelemetry it is critically important to preserve code integrity, including copyright.
+
+Extensions, apps, or bots that can **arbitrarily modify code are NOT allowed**. Read-only code access, opening pull requests from another fork, or bug triage bots can be considered to be installed.
+
+## Requesting installing of Apps, Extensions, or bots
+
+- The default answer for installing third party tools in the org is "NO". Justification needs to be provided and supported by multiple org members.
+- Open an issue at https://github.com/open-telemetry/community/issues
+  - Include reasoning and SIG(s) requesting this
+  - Requests from maintainers typically carry higher weight; please make sure to discuss in the SIG you participate.
+  - List permissions this extension, app, or bot requests.
+  - If possible: point to other uses of this extensions in OSS.
+- GC member needs to approve with no other GC members raising concerns. It is recommended to discuss each extension, app, or bot that is about to be installed at GC meeting for awareness.
+- Once GC approval received, TC member will install the extension.
+
+## Writing your GitHub Actions pipelines
+
+Many GitHub Action workflows do not require a dedicated GitHub account. Good examples are auto-assign workflows for issues and PRs such as [the one used in specifications repo](https://github.com/open-telemetry/opentelemetry-specification/blob/main/.github/workflows/auto-assign-issue.yml).
+
+There are cases when a dedicated account is needed to perform some higher privilege operations. In this cases the recommendation for maintainers is to use their accounts. See [example](https://github.com/open-telemetry/opentelemetry-specification/blob/main/.github/workflows/publish-schemas.yml).
+
+Bot accounts are not recommended, as they cannot sign the CLA. If bot account needs to be used, it should be clear who owns this account and who signed CLA on behalf of this bot.
```

Comment:
there was recent ask in maintainer meeting related to using bot accounts for release automation, and the GC meeting notes from 2021-09-02 mention:

> CNCF easycla team gave permission to exempt specific branches or branch prefixes from the pull request requirement to create/edit branches given that no code on the branch introduces new copywritable content (release automation)

is this something that makes sense to address here?

Reply:
@dyladan maybe you can add information about this as a follow up PR?

Reply:
I'm the person who's been our liaison to the easycla team, I can explain.

```
Bot accounts are not recommended, as they require special work to pass CLA checks. If pushing automatically-generated, non-copyrightable code using a bot account is required, an explanation should be sent to the Governance Committee for review and forwarding to the EasyCLA team to exempt the bot's commits from the CLA requirement.
```

Reply:
```suggestion
Bot accounts are not recommended, as they require special work to pass CLA checks. If pushing automatically-generated, non-copyrightable code using a bot account is required, an explanation should be sent to the Governance Committee for review and forwarding to the EasyCLA team to exempt the bot's commits from the CLA requirement.
```